src/dice/DamageRoll.ts

```diff
@@ -120,11 +139,12 @@ class DamageRoll extends foundry.dice.Roll<DamageRoll.Data> {
 				} else {
 					primaryTerm = new PrimaryDie({
 						number: 1,
-						faces: firstDieTerm.faces,
-					});
+						faces: firstDieTerm.faces ?? 6,
+						modifiers: [],
+					} as object as foundry.dice.terms.Die.TermData);
 
 					// Add rollMode
-					primaryTerm.number = number + Math.abs(rollMode);
+					(primaryTerm as { number: number }).number = (number ?? 1) + Math.abs(rollMode);
```

Comment:
Why is this one different than the one above? Can't we have one type to cover all cases?